gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -701,3 +701,24 @@ def delete_firewall(self, context, firewall, host):
             self._print_exception('Failure', resp.status_code, url,
                                   'create', resp.content)
             return const.STATUS_ERROR
+
+    def configure_healthmonitor(self, context, kwargs):
+        return self._check_vm_health(kwargs)
+
+    def _check_vm_health(self, kwargs):
```

Comment:
I am using some of the kwargs in the LOG of _check_vm_health(). And this log has info about the type of HM being used.Here we need kwargs. We will have to remove the logs in that case. What do u suggest. Also in case if subprocess command fails I am printing the exception. We may loose this exception if make this common function. Plz suggest on this.